### ISSUE TITLE
Vollbild zu gross

### ISSUE THREAD

**Issue by picuogl** [microsoft/vscode#108807]
Issue Type: <b>Bug</b>

Beim Verwenden von zwei Bildschirmen, wir Visual Studio Code, im Vollbild-Modus, zu gross dargestellt. Auf dem zweiten Bildschirm ist der Rahmen von VSC zu sehen.

VS Code version: Code 1.50.1 (d2e414d9e4239a252d1ab117bd7067f125afd80a, 2020-10-13T15:06:15.712Z)
OS version: Windows_NT x64 10.0.19041

**Comment by LennardFi** [comment]
I don't know where to post this. But this is the translation of the issue description above:

> When using two screens, Visual Studio code is displayed too large in full screen mode. On the second screen you can see the frame of VSC.

I encountered the same problem. You can see this in the screenshots I attached. I made a screenshot of the top right edge of my left screen. On the right screen vscode is open. As you can see, the first image contains a one pixel sized border of vscode if i maximized the vscode window. The second image doesn't contain such a border after I reduced the window size or minimized the vscode window.

VS Code opened:

![VS Code opened](https://user-images.githubusercontent.com/43301234/96572989-d10fc780-12cd-11eb-8aac-f808af9899b2.png)

VS Code minimized

![VS Code minimized](https://user-images.githubusercontent.com/43301234/96573891-079a1200-12cf-11eb-9a5b-9225655896b2.png)

**Comment by bpasero** [comment]
I think we might have fixed this with Electron 9.3.2.

Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by LennardFi** [comment]
Yes, already fixed in the newest nightly build. Thx @bpasero 

Version of the tested nightly build:
```
Version: 1.51.0-insider (user setup)
Commit: 13b3c937dc5e3816c79bdd2cdf2cdf6f9c727b75
Date: 2020-10-20T06:07:53.466Z
Electron: 9.3.2
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041
```
